app/presenters/services_presenter.rb

```diff
@@ -22,6 +22,6 @@ class ServicesPresenter < Jsonite
   property :schedule, with: SchedulesPresenter
   property :notes, with: NotesPresenter
   property :categories, with: CategoryPresenter
-  # property :addresses, with: AddressPresenter
+  property :addresses, with: AddressPresenter
```

Comment:
It doesn't seem like the rest of your implementation required this to be uncommented. If that's correct, then could you leave it commented out? I'm not sure why it was commented out in the first place, but it's probably better to leave it commented out if we don't need it and if we're not sure why it was removed.

Reply:
I was thinking that It's probably needed if we want to present a service with addresses but I'll leave it commented out for now.